x-pack/test/lists_api_integration/utils.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SuperTest } from 'supertest';
+import supertestAsPromised from 'supertest-as-promised';
+
+import { ListSchema } from '../../plugins/lists/common';
+import { LIST_INDEX } from '../../plugins/lists/common/constants';
+
+/**
+ * Creates the lists index for use inside of beforeEach blocks of tests
+ * This will retry 20 times before giving up and hopefully still not interfere with other tests
```

Comment:
Just curious - did we find that it took around 20 attempts to ensure the test hit at least once, or is this just more an arbitrary number?

Reply:
Arbitrary number, figured I would increase that default if we hit the errors again which we haven't luckily.